packages/browser-wallet/src/popup/store/utils.ts

```diff
@@ -91,7 +91,7 @@ export function atomWithChromeStorage<V>(key: ChromeStorageKey, fallback: V, wit
 
         const listener = (changes: Record<string, chrome.storage.StorageChange>) =>
             getStoredValue().then((value) => {
-                if (key in changes) {
+                if (key in changes || ChromeStorageKey.NetworkConfiguration in changes) {
```

Comment:
Wait... Why do we getStoredValue and then inside check if the changes has anything to do with the current key?

Reply:
(Instead of the opposite)

Reply:
Yes.. good point...